core/tests/behavior/list.rs

```diff
@@ -98,13 +95,14 @@ pub async fn test_check(op: Operator) -> Result<()> {
 
 /// List dir should return newly created file.
 pub async fn test_list_dir(op: Operator) -> Result<()> {
-    let path = uuid::Uuid::new_v4().to_string();
+    let parent = uuid::Uuid::new_v4().to_string();
+    let path = format!("{parent}/{}", uuid::Uuid::new_v4().to_string());
```

Comment:
It's better for cleaning the parent dir?

Same for other tests.

Reply:
We can do it in another PR.